Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [workspace]
 members = [
     "bin/trin", 
-    "ethportal-api",
+    "crates/ethportal-api",
+    "crates/e2store",
+    "crates/light-client",
+    "crates/metrics",
+    "crates/rpc",
+    "crates/beacon",
+    "crates/evm",
+    "crates/history",
```

Comment:
*nitpick* ... maybe we could leave the prefix on the `history` / `state` / `beacon` crates? Sometimes when you have to make a change to one of these, you typically have to replicate it across all three, so it's nice when they're grouped together alphabetically

Reply:
I was thinking of putting them in a folder
so
``crates/subnetworks/history``
``crates/subnetworks/state``
``crates/subnetworks/history``

What do you think? I am open to ideas, I am fine doing the suggestion above, I think a folder would be a little cleaner though

Reply:
yeah, works for me!